app/views/layouts/_content.html.haml

```diff
@@ -63,6 +63,9 @@
             = render :partial => "layouts/listnav"
           = yield :left
         #custom_left_cell
+- elsif @angular_app_name
+  .container-fluid{:style => "overflow: hidden; height: 100%;", 'data-ng-app' => "#{@angular_app_name}"}
+    %div{'ui-view' => ''}
 - else
```

Comment:
`_content` is definitely too complex already, but OTOH it is *the* right side, right? But then again, on the *other* other hand, if we're doing `ui-router`, we're no longer *just* on the right side. But.. uh, otoooh, the left side is always just listnav or explorer or empty, which should be controller by the *outer* angular bits, which may not necessarily be the same module as the inner ones .. so, yup, we'll see indeed :).